src/ert/config/queue_config.py

```diff
@@ -20,11 +19,53 @@
 
 GENERIC_QUEUE_OPTIONS: List[str] = ["MAX_RUNNING"]
 VALID_QUEUE_OPTIONS: Dict[Any, List[str]] = {
-    QueueSystem.TORQUE: _clib.torque_driver.TORQUE_DRIVER_OPTIONS
-    + GENERIC_QUEUE_OPTIONS,
-    QueueSystem.LOCAL: [] + GENERIC_QUEUE_OPTIONS,  # No specific options in driver
-    QueueSystem.SLURM: _clib.slurm_driver.SLURM_DRIVER_OPTIONS + GENERIC_QUEUE_OPTIONS,
-    QueueSystem.LSF: _clib.lsf_driver.LSF_DRIVER_OPTIONS + GENERIC_QUEUE_OPTIONS,
+    QueueSystem.TORQUE: GENERIC_QUEUE_OPTIONS
+    + [
+        "CLUSTER_LABEL",
+        "DEBUG_OUTPUT",
```

Comment:
Deprecated I think, as for LSF

Reply:
I guess we need them to avoid the parser crashing heavily on deprecated terms?